pyaedt/modules/PostProcessor.py

```diff
@@ -3834,8 +3836,8 @@ def export_model_obj(self, assignment=None, export_path=None, export_as_single_o
             self._app.modeler.oeditor.ExportModelMeshToFile(fname, assignment)
             return [[fname, "aquamarine", 0.3]]
 
-    @pyaedt_function_handler()
-    def export_mesh_obj(self, setup_name=None, intrinsics=None):
+    @pyaedt_function_handler(setup_name="setup")
+    def export_mesh_obj(self, setup=None, intrinsics=None):
         """Export the mesh in ``aedtplt`` format.
         The mesh has to be available in the selected setup.
         If a parametric model is provided user can choose the mesh to export providing a specific set of variations.
```

Comment:
```suggestion
        If a parametric model is provided, you can choose the mesh to export by providing a specific set of variations.
```
Avoid "user" and use "you" instead (unless distinguishing between an administrator or a developer versus a user is necessary).